index.html

```diff
@@ -0,0 +1,352 @@
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="header-main_container">
+    <div class="header-leftpan">
+      <div class="header-logo">
+        <a href="/" class="flipkart-logo">
+          <img width="75px"
+            src="https://static-assets-web.flixcart.com/fk-p-linchpin-web/fk-cp-zion/img/flipkart-plus_8d85f4.png"
+            alt="Flipkart" class="flipkart-logo-image" />
+        </a>
+        <a href="https://www.flipkart.com/plus" class="flipkart-prime">
+          Explore <span class="plus">Plus</span><img src="https://static-assets-web.flixcart.com/fk-p-linchpin-web/fk-cp-zion/img/plus_aef861.png"
+            width="10">
+        </a>
+      </div>
+      <div class="header-searchbar">
+        <form class="header-form-search">
+          <input type="text" title="Search for products, brands and more"
+          placeholder="Search for products, brands and more">
+          <span class="material-symbols-outlined search-icon">search</span>
+        </form>
+      </div>
+    </div>
+    <div class="header-rightpan">
+      <div class="header-login">
+        <a class="login-button">Login</a>
+      </div>
+      <div class="header-more">
+        <div class="more">More
+          <span class="material-symbols-outlined">expand_more</span>
+        </div>
+        <div class="more-content">
+          <a href="#"><span class="material-symbols-outlined">
+            notifications
+            </span><div class="option-name">
+              Notification Preference
+            </div>
+          </a>
+          <a href="#"><span class="material-symbols-outlined">
+            live_help
+            </span> <div class="option-name">
+              24x7 Customer Care
+            </div>
+          </a>
+          <a href="#"><span class="material-symbols-outlined">
+            moving
+            </span>
+            <div class="option-name">
+              Advertise
+            </div>
+          </a>
+          <a href="#"><span class="material-symbols-outlined">
+            download
+            </span>
+            <div class="option-name">
+              Download App
+            </div>
+          </a>
+        </div>
+      </div>
+      <div class="header-cart">
+        <span class="material-symbols-outlined cart-icon">
+          shopping_cart
+          </span>
+        <span class="cart-text">Cart</span>
+      </div>
+    </div>
+  </div>
+  <div class="category-pannel">
+    <div class="category-items">
+      <div class="item">
+        <div class="item1-image">
+          <img src="https://rukminim1.flixcart.com/flap/128/128/image/f15c02bfeb02d15d.png?q=100" alt="Top Offers" />
+        </div>
+        <div class="item1-name">Top Offers
```

Comment:
`item1-name` is missing in `.css` file. What css properties are they adding to this `div`.